packages/teams-js/src/internal/utils.ts

```diff
@@ -429,6 +429,12 @@ export function validateUrl(url: URL, errorToThrow?: Error): void {
   }
 }
 
+export function fullyQualifyUrlString(urlString: string): URL {
```

Comment:
>fullyQualifyUrlString [](http://example.com/codeflow?start=16&length=21)

Can you add a comment describing what this function does with an example? I don't think it's obvious to people who haven't been staring at this code for a while why it's creating a new DOM element and what goal that achieves.